src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int ROOF_OF_NAMES = 2;
+    private static final int INDEX_OF_VALUES = 1;
+    private static final int INDEX_OF_SUPPLY_VALUE = 7;
+    private static final int INDEX_OF_BUY_VALUE = 4;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder statisticBuilder = new StringBuilder();
+        String textFromFile = readFromFile(fromFileName);
+        String[] dataArray = textFromFile.split(System.lineSeparator());
+        int totalValue;
+        for (int i = 0; i < ROOF_OF_NAMES; i++) {
+            if (i == 0) {
+                statisticBuilder.append("supply,");
+            } else {
+                statisticBuilder.append("buy,");
+            }
```

Comment:
append all data together only after you count everything. Also, you should have 2 different methods: one for counting, and another for creating the report.